brainscore_vision/metrics/regression_correlation/__init__.py

```diff
@@ -11,6 +11,15 @@
 metric_registry['linear_predictivity'] = lambda *args, **kwargs: CrossRegressedCorrelation(
     regression=linear_regression(), correlation=pearsonr_correlation(), *args, **kwargs)
 
+# temporal metrics
+from .metric import SpanTimeCrossRegressedCorrelation
+
+metric_registry['spantime_pls'] = lambda *args, **kwargs: SpanTimeCrossRegressedCorrelation(
+  regression=pls_regression(), correlation=pearsonr_correlation(), *args, **kwargs)
+metric_registry['spantime_ridge'] = lambda *args, **kwargs: SpanTimeCrossRegressedCorrelation(
+  regression=ridge_regression(), correlation=pearsonr_correlation(), *args, **kwargs)
```

Comment:
not just `temporal_ridge`?

Reply:
Because "spantime" and "pertime" are essentially different: the former shares the weights across time and the latter train separate weights for each time bin.